pkg/manager/risingwave_controller_manager_impl.go

```diff
@@ -58,9 +58,35 @@ type risingWaveControllerManagerImpl struct {
 	forceUpdateEnabled bool
 }
 
+func getStandaloneStatusUtil(rw *risingwavev1alpha1.RisingWave, logger logr.Logger, readyReplicas int32, statefulSetName string) risingwavev1alpha1.ComponentReplicasStatus {
+	requiredReplicas := rw.Spec.Components.Standalone.Replicas
```

Comment:
`rw.Spec.Components.Standalone` could be nil. I think it's the reason that E2E failed.